src/patternfly/components/Button/button.scss

```
@@ -317,7 +316,7 @@
     --#{$button}--m-clicked__icon--Color: var(--#{$button}--m-primary--m-clicked__icon--Color);
 
     .#{$badge}.pf-m-unread::after {
```

I'd probably take the `::after` off, I don't think it's needed.

```suggestion
    .#{$badge}.pf-m-unread {
```